template/web/src/containers/Head.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { useStaticQuery, graphql } from 'gatsby'
 import Helmet, { HelmetProps } from 'react-helmet'
 // import { mapToSingleObject } from '@/lib/helpers'
-import fonts from '@/assets/fonts.css'
+// import fonts from '@/assets/fonts.css'
```

Comment:
how do we want to deal with fonts import? how did we do in arys after all?

Reply:
In Arys werden sie so importiert:  import '@/assets/fonts.css'